shared/cfg/guest-os/Windows.cfg

```diff
@@ -138,6 +138,8 @@
             time_command = "echo TIME: %date% %time%"
             time_filter_re = "(?<=TIME: \w\w\w ).{19}(?=\.\d\d)"
             time_format = "%m/%d/%Y %H:%M:%S"
+    timedrift.with_pvclock.shared_ntp_date.ntp.with_stop.base_date:
+        rtc_base = 2006-6-17
```

Comment:
Base on the new change in https://github.com/autotest/tp-qemu/pull/1019/files
timedrift.with_pvclock.with_signal_stop.base_date  also need this change.